nifi/nifi-nar-bundles/nifi-kite-bundle/pom.xml

```diff
@@ -0,0 +1,60 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  -->
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.nifi</groupId>
+    <artifactId>nifi-nar-bundles</artifactId>
+    <version>0.0.1-incubating-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>nifi-kite-bundle</artifactId>
+  <version>0.0.1-incubating-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Kite Bundle</name>
+  <description>A bundle of processors that use Kite to store data in Hadoop</description>
+
+  <modules>
+    <module>nifi-kite-processors</module>
+    <module>nifi-kite-nar</module>
+  </modules>
+  
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <configuration>
+            <redirectTestOutputToFile>true</redirectTestOutputToFile>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.apache.nifi</groupId>
+        <artifactId>nifi-kite-processors</artifactId>
+        <version>0.0.1-incubating-SNAPSHOT</version>
```

Comment:
This gets updated automatically, so I don't see a problem keeping it. Plus, maven is terrible at consistent property values when resolving POM dependencies, so I would like to keep these. Joey originally pointed this out and I changed it to this.